operator/v1/types_dns.go

```diff
@@ -287,6 +287,24 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+
+	// protocolStrategy specifies the protocol to use for upstream DNS
+	// requests. Valid values for protocolStrategy are ForceTCP or omitted.
+	// When omitted, that means no opinion, and the platform uses a default
+	// behavior, which is subject to change over time. Currently, the
+	// default behavior is to use the protocol of the original client
+	// request as the upstream protocol. ForceTCP specifies that the
+	// platform should use TCP for all upstream DNS requests, even if the
+	// client request uses UDP. ForceTCP is useful for UDP-specific issues
+	// such as those created by non-compliant upstream resolvers or UDP
+	// stability problems, but may consume more bandwidth or increase DNS
+	// response time. Note that protocolStrategy only affects the protocol of
+	// DNS requests that CoreDNS makes to upstream resolvers. It does not
+	// affect the protocol of DNS requests between clients and CoreDNS.
+	//
+	// +optional
+	protocolStrategy protocolStrategy `json:"protocolStrategy,omitempty"`
```

Comment:
```suggestion
	ProtocolStrategy ProtocolStrategy `json:"protocolStrategy,omitempty"`
```
I think it's a typo, I saw some comments were they were exported.

Reply:
Ah whoops, I did a copy/replace but it was case insensitive.